dotcom-rendering/src/web/server/articleToHtml.tsx

```diff
@@ -173,8 +174,27 @@ export const articleToHtml = ({ article }: Props): string => {
 		return `https://amp.theguardian.com/${article.pageId}`;
 	};
 
+	// I would have like to have had this closer to the implementation
+	// but we don't have `pageId` available in `pageTemplate.ts`, so this
+	// was as close as I could get it. It's a test and will be removed anywho.
+
+	// We are only including recently published articles to help with data normalisation
+	// hopefully we merge soon so we don't need to keep changing this.
+	const isNewArticle =
+		new Date(article.webPublicationDate) > new Date('2023-03-29');
+
+	// 0-5: control - don't disable link
+	// 6-11: variable - disable link
+	const disableAmpLinkGroup = getGroup(article.pageId);
```

Comment:
Is this deterministic? (e.g same pageID always = same group number)

Reply:
Indeed it is and we have [a test](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/amp/components/ContentABTest.test.tsx#L8-L39) to prove it.